sources/org.osbuild.containers-storage

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
```

Comment:
Typo in the commit message `/etc/containers/storage/conf` should be `/etc/containers/storage.conf`